docs/examples/transformations.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Transformations
+parent: Examples
+nav_order: 4
+---
+# Transformations
+
+With transformations we can manipulate data prior to writing to the data store.
+
+## Aggregate
+With aggregates we can take multiple records and output a single record.
+
+```csharp
+using (var writer = new BulkWriter<int>(connectionString))
+{
+    var items = Enumerable.Range(1, 1000).Select(i => new MyClass { Id = i, Name = "Bob" });
+    var pipeline = EtlPipeline.StartWith(items)
+        .Aggregate(f => f.Sum(c => c.Id))
+        .WriteTo(writer);
+
+    await pipeline.ExecuteAsync();
+}
+```
+
```

Comment:
Show expected output (in abbreviated form where necessary) so people don't actually have to run the code to really see what it's doing.